### ISSUE TITLE
Leave this page confirmation without changing content

### ISSUE THREAD

**Issue by confluxgroup** [supercool/Pimp-My-Matrix#35]
I've noticed that Craft's "any changes will be lost if you leave this page" dialog comes up on entries that have blocks with Pimp My Matrix tabs enabled even if nothing has been changed.

From my testing, it appears that this happens only after you change tabs on one of the blocks. 

Is there any way to reset the flag that triggers that dialog after changing tabs?

**Comment by joshangell** [comment]
I'm not totally sure there is a way of re-setting that reliably, more desirable would be to suppress the clicks from the tabs from bubbling up to whatever is listening that triggers that alert.

I'll certainly take a look when I next get round to working on this :smile:

**Comment by marcato15** [comment]
Checking in on this feature request. Looks like you removed the assignment. Is that because you're unable to work on it any more? Just checking b/c this would prevent us from using the plugin. If you can't maintain it then we'll look at either forking and submitting a PR or just not using it.

**Comment by joshangell** [comment]
@marcato15 Yes I’m afraid I am unable to work on this project any more due to other work commitments. If its a simple PR with no obvious conflicts then @Nav33d may consider merging it 🙂

**Comment by marcato15** [comment]
Sounds good.

**Comment by Nav33d** [comment]
@marcato15 PRs are always welcome :)

One thing to keep in mind that we are no longer supporting this plugin and won't be porting it over to Craft 3
